guides/common/modules/proc_troubleshooting-dhcp-problems.adoc

```diff
@@ -1,5 +1,5 @@
-[id="Troubleshooting_DHCP_Problems_{context}"]
-= Troubleshooting DHCP problems in {Project}
+[id="Troubleshooting_DHCP_Problems"]
```

Comment:
```suggestion
[id="troubleshooting-dhcp-problems"]
```

If we change the anchor, then we should grab this opportunity and align it with the file name :) Makes grepping much more pleasurable.